impress-org/givewp
# Form field manager custom fields rendered with field api look good in multi-step form template

Resolves https://github.com/impress-org/givewp/issues/5945

## Description


FFM custom fields look good with [new field API implementation](https://github.com/impress-org/give-form-field-manager/pull/356) and exiting field logic in the multi-step form template.

## Affects


These pull request changes affect FFM custom field output on donation form in multi-step form template

## Testing Instructions


- Check ffm custom field with multi-step form template with [field API implementation](https://github.com/impress-org/give-form-field-manager/pull/356) and exiting ffm version.

## Pre-review Checklist
